src/main/java/org/jabref/gui/preferences/FileTab.fxml

```diff
@@ -31,7 +32,7 @@
         <HBox alignment="CENTER_LEFT" spacing="10.0">
             <children>
                 <RadioButton fx:id="resolveStringsAll" alignment="TOP_LEFT" maxWidth="Infinity"
-                             text="Resolve strings for all fields except" toggleGroup="$stringsResolveToggleGroup"/>
+                             text="%Resolve strings for all fields except" toggleGroup="$stringsResolveToggleGroup"/>
```

Comment:
Tip I just discovered: In scene builder you have to click on the right side of the textfield the little icon,  Replace with international strings and then you can paste the text and it will automatically add the %sign